### PR TITLE
[Create survey] [Setting pop-up][Manager tab] Should DISPLAY error me…

### PR DESCRIPTION
**Pull request by nampx-0860**
Summary : Should DISPLAY message "This email is not an account" when enter an email NOT EXIST in system to manage survey
Precondition: Have an email not exist in system ("silverbullet2812@gmail.com")
Step to reproduce:
1. Open create survey form.
2. Enter valid data into create survey form.
3. Click on button "Send" in "Menu taskbar" 
4. On tab "Setting pop-up", open "Manager tab" 
5. Enter: "silverbullet2812@gmail.com" 
6. Press "Enter".
Actual result : Add manager email successfully
Expected result : Can not allow add an email is not an account in system to manager survey and display msg "This email is not an account in system"

### CODE REVIEW

**Comment by nampx-0860** [comment]
input_message
https://edu-redmine.sun-asterisk.vn/issues/11144

**Comment by nampx-0860** [comment]
fixnewbug
https://edu-redmine.sun-asterisk.vn/issues/11144

**Comment by tranducquoc** [comment]
anh copy 1 list email vào đây thì thế nào?

**Comment by nampx-0860** [comment]
cái này nhập 1 lần 1 mail nên k copy 1 list vào dc anh ạ

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
Comment (tranducquoc) [inline npm-debug.log]: file này đẩy lên làm gì. cho vào gitignore đi em

**Pull request merged**